Makefile

```diff
@@ -52,11 +52,13 @@ benchmark:
 		swift-composable-architecture-benchmark
 
 format:
-	swift format \
+	@mv ./Sources/ComposableArchitecture/Documentation.docc ./tmp.docc
+	@swift format \
 		--ignore-unparsable-files \
 		--in-place \
 		--recursive \
 		./Examples ./Package.swift ./Sources ./Tests
+	@mv ./tmp.docc ./Sources/ComposableArchitecture/Documentation.docc
```

Comment:
this is the best I could come up with to make swift-format not change the tutorial snippet files. is there something better we can do?